New version: BioMarkovChains v0.8.1

JuliaRegistrator opened the pull request:
- Registering package: BioMarkovChains
- Repository: https://github.com/camilogarciabotero/BioMarkovChains.jl
- Created by: @camilogarciabotero
- Version: v0.8.1
- Commit: 056e573b561c3e026866e33c09bcc11740a17ff0
- Reviewed by: @camilogarciabotero
- Reference: https://github.com/camilogarciabotero/BioMarkovChains.jl/commit/056e573b561c3e026866e33c09bcc11740a17ff0#commitcomment-135613669
- Description: Representing biological sequences as Markov chains
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd63a2096f04b78f6b6264a805c70e0043716c701806c0d789453ff258c40fa8c2eac3ab516ddfb8177cb65bf842d10a8294654d4aabcdb24b58377456fc1c05c83a68349960c0b0c5db95cefb0860bfc28ea545cfa633dff59b97216d62b4e6fb8bf92da3b07262a24faef6f749b77c60f638d375d70eee7798e04eeaf7700c7ff384392c05f415f2cdae7446ce0cd7dc2f42c90e38e83eab4d220d75dad18ae39c57ca7ea997920442565cdf41b792efe57796af5618fe9bde34d74643e685195 -->

camilogarciabotero commented:
[noblock] Is there something I can do to fix the auto-merging problem?

ericphanson commented:
[noblock] you need to specify the minor version [here](https://github.com/camilogarciabotero/BioMarkovChains.jl/blob/d6f9df509715964f03d78589463a04a3c292d6ec/Project.toml#L30) and retrigger registration

JuliaRegistrator committed:
New version: BioMarkovChains v0.8.1

UUID: f861b655-cb5f-42ce-b66a-341b542d4f2c
Repo: https://github.com/camilogarciabotero/BioMarkovChains.jl.git
Tree: fe2c5eb2818faa1b4f3db9bd8e32af6eb17478b6

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
